kubetest/eks.go

```diff
@@ -1,332 +0,0 @@
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-// Package main / eks.go implements kubetest deployer interface for EKS.
-// It uses 'aws-k8s-tester' and 'kubectl' binaries, rather than importing internal packages.
-// All underlying implementation and external dependencies are compiled into one binary.
-package main
-
-import (
-	"flag"
-	"fmt"
-	"io/ioutil"
-	"net/http"
-	"os"
-	osexec "os/exec"
-	"path/filepath"
-	"syscall"
-	"time"
-
-	"github.com/aws/aws-k8s-tester/eksconfig"
-	"github.com/aws/aws-k8s-tester/ekstester"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/test-infra/kubetest/process"
-	"k8s.io/test-infra/kubetest/util"
-)
-
-var (
-	eksKubectlPath      = flag.String("eks-kubectl-path", "/tmp/aws-k8s-tester/kubectl", "(eks only) Path to the kubectl binary to use.")
-	eksKubecfgPath      = flag.String("eks-kubeconfig-path", "/tmp/aws-k8s-tester/kubeconfig", "(eks only) Path to the kubeconfig file to use.")
-	eksNodes            = flag.String("eks-nodes", "1", "(eks only) Number of nodes in the EKS cluster.")
-	eksNodeInstanceType = flag.String("eks-node-instance-type", "m3.xlarge", "(eks only) Instance type to use for nodes.")
-)
-
-func migrateEKSOptions() error {
-	// Prevent ginkgo-e2e.sh from using the cluster/eks functions.
-	if err := os.Setenv("KUBERNETES_CONFORMANCE_TEST", "yes"); err != nil {
-		return err
-	}
-	if err := os.Setenv("KUBERNETES_CONFORMANCE_PROVIDER", "eks"); err != nil {
-		return err
-	}
-	return util.MigrateOptions([]util.MigratedOption{
-		// Env vars required by upstream ginkgo-e2e.sh.
-		{
-			Env:    "KUBECTL",
-			Option: eksKubectlPath,
-			Name:   "--eks-kubectl-path",
-		},
-		{
-			Env:    "KUBECONFIG",
-			Option: eksKubecfgPath,
-			Name:   "--eks-kubeconfig-path",
-		},
-		// Env vars specific to aws-k8s-tester.
-		{
-			Env:    "AWS_K8S_TESTER_EKS_WORKER_NODE_ASG_MIN",
-			Option: eksNodes,
-			Name:   "--eks-nodes",
-		},
-		{
-			Env:    "AWS_K8S_TESTER_EKS_WORKER_NODE_ASG_MAX",
-			Option: eksNodes,
-			Name:   "--eks-nodes",
-		},
-		{
-			Env:    "AWS_K8S_TESTER_EKS_WORKER_NODE_INSTANCE_TYPE",
-			Option: eksNodeInstanceType,
-			Name:   "--eks-node-instance-type",
-		},
-	})
-}
-
-// eksDeployer implements EKS deployer interface using "aws-k8s-tester" binary.
-// Satisfies "k8s.io/test-infra/kubetest/main.go" 'deployer' and 'publisher" interfaces.
-// Reference https://github.com/kubernetes/test-infra/blob/master/kubetest/main.go.
-type eksDeployer struct {
-	stopc chan struct{}
-	cfg   *eksconfig.Config
-	ctrl  *process.Control
-}
-
-// newEKS creates a new EKS deployer.
-func newEKS(timeout time.Duration, verbose bool) (ekstester.Deployer, error) {
```

Comment:
looks like kubetest main uses this

https://github.com/kubernetes/test-infra/blob/e86e60c9c133773a78093ce5e6eafa16c5edb6ff/kubetest/main.go#L243-L244

maybe also leave deletion of kubetest1 eks as a follow PR?

Reply:
oh, just fixed it. We are ok to remove kubetest1 now, since it's broken.